### DISCUSSION TITLE
WROOM32EU GPIO9 and GPIO10 init UART1

### DISCUSSION THREAD

**Discussion by allisyuxianshen** [micropython/micropython#14184] [ESP32]
### Checks

- [X] I agree to follow the MicroPython [Code of Conduct](https://github.com/micropython/micropython/blob/master/CODEOFCONDUCT.md) to ensure a safe and respectful space for everyone.

- [X] I've searched for [existing issues](https://github.com/micropython/micropython/issues) regarding this feature, and didn't find any.

### Description

I wanted to use GPIO9 and GPIO10 on the WROOM32EU as UART1. However, after checking some forums, I found out that these pins are used for flash memory in this version, but the official documentation mentions that these two pins can still be used as UART. I tried to implement this in Arduino and it worked, but I couldn't achieve the same with MicroPython. Is there any way to do this? 

### Implementation

- [X] I intend to implement this feature and would submit a Pull Request if desirable.
- [X] I hope the MicroPython maintainers or community will implement this feature.
- [ ] I would like to [Sponsor](https://github.com/sponsors/micropython#sponsors) development of this feature.

**Comment by davefes** [comment]
But the docs also say:

> The ESP32 has three hardware UARTs: UART0, UART1 and UART2. They each have default GPIO assigned to them, **however depending on your ESP32 variant and board, these pins may conflict with embedded flash, onboard PSRAM or peripherals.**

My question would be ... how does Arduino work-around this apparent conflict?

**Comment by allisyuxianshen** [comment]
Hello davefes

Right, so I later changed the pin configuration to other GPIOs. Thank you.
